src/components/util/util.test.js

```diff
@@ -366,6 +380,30 @@ describe('getContributors', () => {
   });
 });
 
+describe('isConsortiumEnabled', () => {
+  it('should return false', () => {
+    const data = isConsortiumEnabled();
+    expect(data).toBeFalsy();
+  });
```

Comment:
Under what circumstances should it return false? This makes it sound like the function should _always_ return false. Describe the situation you are actually testing. In this case, sth like "returns false (default) when given no data".